src/migrations/20180516164000-endpoint-specific-apps.js

```diff
@@ -2,12 +2,12 @@ const fs = require('fs');
 const configJsonPath = process.env.CONFIG_MOUNT_POINT;
 
 exports.up = function(knex, Promise) {
-	return new Promise((resolve, reject) => {
+	return new Promise((resolve) => {
 		if (!configJsonPath) {
 			console.log(
 				'Unable to locate config.json! Things may fail unexpectedly!',
 			);
-			resolve({});
+			return resolve({});
```

Comment:
I don't know what would have happened but previously this fell through the `fs.readFile` with an empty path (typescript caught this)